### PR TITLE
Community Assessment - CityTechSoc Spring 

### PR DESCRIPTION
**Pull request by abdulajet**

### CODE REVIEW

File: communities/citytechsocspring/community-assessment.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+- Answers to these questions for current community
+- Answers to these questions for **an ideal version** of your current community
+- Ideas on how to improve in each category
+
+After you've completed these questions, define your community in several short paragraphs for discussion in week 2.
+
+### Definition
+- How would you define your community? Try to be open minded, but here are some examples of ways you could define your community: 
+
+City Tech Society is a community of students at City, University of London who are interested in technology who are looking to learn more about technology or need help with topics covered on the Computer Science course.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: Concrete!

File: communities/citytechsocspring/community-assessment.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  
+### Size
+- How many people are currently in your community?
+
+We have a 220 person mailing list of interested people but active members is around 20-30 but this can vary depending on the time of year.
+- How many would you _like_ to be in your community?
+
+Ideally I would like for the community to have 50 or so active and contributing members all year round.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: Very achievable and realistic increase

File: communities/citytechsocspring/community-assessment.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+Historically there have been similar groups on campus but they folded as soon as the individual running it has graduated.
+
+### Resources
+- What is currently available to people in your community to learn about tech?
+
+Being is London provides quite a vast amount of resources for people to learn more about tech and one of our main aims is to help our members get more information about them and hopefully attend some that may interest them.
+- Are there meetups, seminars, online tools, office hours, hackathons?
+
+Apart from the actual Computer Science course there are countless external meetups for people to attend. Our university's careers and entrepreneurship departments host seminars very regularly. Every Monday we meet up to discuss tech, play games and have a small workshop session. Hackathons are also another focus of our community, we attend many across the London and sometimes in other parts of the UK. We also hosted our own hackathon, HackCity this year.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: The external communities offer a great opportunity for you folks!

File: communities/citytechsocspring/community-assessment.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+Yes
+- Are they trying to ramp-up to get a job?
+
+Members on from the Computer Science course do sometimes use the community to learn more about roles in tech and ask questions from older members.
+- Are they passionate about developing, regardless of their major?
+
+The students that we see from non CS degrees tend to be more passionate about developing whereas some of our CS students fail to make the distinction between coursework and working on their own projects and do sometimes see developing as a chore.
+- What activities and goals can you set for your community based on their existing goals?
+
+We are looking into working on a project together as a community, whether it be a website or game. This will be inviting to members who want to develop more as well as CS students who want to work on their portfolio.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: Great! Is this what your impact proposal will be based on?

File: communities/citytechsocspring/community-assessment.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+People get involved in our community because they want to learn more about tech / CS, or they will like more support with the Computer Science course. 
+- Are they students hoping to get a degree?
+
+Yes
+- Are they trying to ramp-up to get a job?
+
+Members on from the Computer Science course do sometimes use the community to learn more about roles in tech and ask questions from older members.
+- Are they passionate about developing, regardless of their major?
+
+The students that we see from non CS degrees tend to be more passionate about developing whereas some of our CS students fail to make the distinction between coursework and working on their own projects and do sometimes see developing as a chore.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: What can you do to address this?

Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: Or encourage and support it, on the inverse!

File: communities/citytechsocspring/community-assessment.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+We tend to use classrooms which have more comfortable seating and a better set up for our events compared to a lecture theatre.
+- Is there water or snacks available nearby?
+
+Water is available nearby on campus and there are shops very close to the university. We also bring a small number of snacks to the meetups when we can and cater for larger events such as hack days. 
+
+### Communication
+- How does your community get in touch with one another?
+
+Right now we use the mailing list and slack for communication.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: How does slack work out for you?

File: communities/citytechsocspring/community-assessment.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+### Communication
+- How does your community get in touch with one another?
+
+Right now we use the mailing list and slack for communication. 
+- Is there an email group, a GitHub organization, a Twitter account, a Facebook group, a chat room or channel?
+
+We have a GitHub org for committee members to manage the website. We have a public Twitter and Facebook page for announcements to external people and a currently inactive Facebook group.
+- Could your organization benefit from one or more of those, or another way of communicating?
+
+We are in discussions of a platform to move onto from Slack to get more engagement from members.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: Ah, I guess this answers the above!

File: communities/citytechsocspring/community-assessment.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+We have a GitHub org for committee members to manage the website. We have a public Twitter and Facebook page for announcements to external people and a currently inactive Facebook group.
+- Could your organization benefit from one or more of those, or another way of communicating?
+
+We are in discussions of a platform to move onto from Slack to get more engagement from members. 
+
+### Culture
+- How welcoming is your community to new people and new ideas?
+
+We try to be very welcoming; we tend to keep a flat structure as far as members are concerned so nobody is worried about speaking their mind.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: How do you reflect this to the members? Do they know they can speak their mind?

File: communities/citytechsocspring/community-assessment.md [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+- What happens if there is a disagreement?
+
+Disagreements are usually very trivial but myself or one of the committee members will step in to moderate. Things can also be escalated to either the Department of Computer Science or the Students Union.
+- Are there any policies or procedures in place, and are they documented?
+
+As a part of the Students Union they have policies and procedures set in place to follow in case of any disagreements. We also have a constitution that outlines our goals, structure and procedures.
```
Comment (joenash) [inline communities/citytechsocspring/community-assessment.md]: Have you considered a code of conduct?

Comment (abdulajet) [inline communities/citytechsocspring/community-assessment.md]: This is something we could work on.

File: communities/citytechsocspring/community-assessment.md (hunk 8 above)
Comment (abdulajet) [inline communities/citytechsocspring/community-assessment.md]: We tend to ask for feedback from members at the beginning of weekly meetups. Along the lines of what sort of activities do you want to see, what could use improvement etc.

**Pull request merged**